### DISCUSSION TITLE
how to add channel of app's content to android tv homescreen

### DISCUSSION THREAD

**Discussion by coucoseth** [react-native-tvos/react-native-tvos#530] [Q&A] [answered]
How can my app show content on android tv home just like netflix and youtube showing suggestions
![IMG_9562 2](https://github.com/react-native-tvos/react-native-tvos/assets/45995143/476d891a-bd88-40d0-9a14-125b1bfa7dba)

**Comment by wouterds** [comment, marked as answer]
You can do this using the Android TV provider API, it's not part of React Native so you will have to write custom Java/Kotlin logic for this. But it is definitely possible to incorporate this native bit in a React Native app.

- https://developer.android.com/training/tv/discovery/recommendations
- https://developer.android.com/training/tv/discovery/recommendations-channel

**Comment by coucoseth** [comment]
i have tried to follow the documentation but my very little knowledge of Java/Kotlin isnt getting me to the desired output. Anyone with an implimentation in react native would have saved me. kindly post it here. @wouterds
